### ISSUE TITLE
Attach tarballs and other artifacts to Github releases

### ISSUE THREAD

**Issue by tabacco** [ImageMagick/ImageMagick#7493]
### Describe the solution you'd like

I'd like to see the tarballs and other release aritfacts (for example, shasum files) available from imagemagick.org/archive attached to the releases in Github as well. It'd make Github another mirror of the downloads, and it's often the first place people look for release artifacts for open source projects.

### Describe alternatives you've considered

The existing automated .tar.gz downloads on releases aren't guaranteed to have stable sha sums by Github, so they're not suitable for download in pipelines that need to verify the downloaded tarball.

**Comment by kphatak** [comment]
Website is down for more than 24 hours!!!

**Comment by fmw42** [comment]
It is being worked on. Sorry for the delay.  Please be patient

**Comment by kphatak** [comment]
Thank you for the update. Appreciate it.

**Comment by tabacco** [comment]
Yeah just to be clear, my ask here was originally inspired by that outage, but not directly related to it. It'd be great to have a mirror of the builds here on Github, regardless of the state of imagemagick.org

**Comment by urban-warrior** [comment]
We have begun the process of moving everything ImageMagick to Github.   As you can surmise, this process is onerous and will take time to get it working efficiently.

**Comment by kmarekspartz** [comment]
These appear to now be here: https://github.com/ImageMagick/ImageMagick/releases

This likely can be closed

**Comment by dlemstra** [comment]
Not everything has been moved yet but we have made some steps.
